src/UI/examples/Button/Standard/with_js_binding.php

```diff
@@ -0,0 +1,14 @@
+<?php
+function with_js_binding() {
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    return $renderer->render(
+		$f->button()->standard("Goto ILIAS", "#")
```

Comment:
Argl, it is even worse. Removing href completely is not an option due to accessability reasons. <a> elements without href are not accessible by keyboard, which makes them completely inaccessible to users without mouse or touch device. Observable in the trigger of the awereness tool in the top toolbar. I do not think we should repeat this pattern.

For the moment my vote therefore goes to 2b: leaving it for the moment to the consumer to fix this in the JS, but not by removing href, but by returning false in the onclick event (and so preventing the default behaviour for this event).